grin/src/Pipeline/Pipeline.hs

```diff
@@ -277,6 +282,8 @@ data PState = PState
     , _psCByResult      :: Maybe CBy.CByResult
     , _psLVAProgram     :: Maybe (IR.AbstractProgram, LVA.LVAMapping)
     , _psLVAResult      :: Maybe LVA.LVAResult
+    , _psETProgram      :: Maybe (IR.AbstractProgram, ET.ETMapping)
```

Comment:
We need to improve the situation here, in a subsequent PR. See [Related card](https://github.com/grin-compiler/grin/projects/1#card-23109370 )

Reply:
Agreed.